lib/promoted/ruby/client.rb

```diff
@@ -57,8 +57,16 @@ def initialize (params={})
           @metrics_timeout_millis  = params[:metrics_timeout_millis] || DEFAULT_METRICS_TIMEOUT_MILLIS
 
           @http_client = FaradayHTTPClient.new
-          @pool = Concurrent::CachedThreadPool.new
           @validator = Promoted::Ruby::Client::Validator.new
+
+          # Thread pool to process delivery of shadow traffic. Will silently drop exccess requests beyond the queue
```

Comment:
nit - s/exccess/excess/

Reply:
I should switch to a theme that puts comments in a lighter color on sunny days :)